plugins/3scale-backend/README.md

```diff
@@ -34,32 +34,105 @@ yarn workspace backend add @janus-idp/backstage-plugin-3scale-backend
-     builder.addEntityProvider(
-       ThreeScaleApiEntityProvider.fromConfig(env.config, {
-         logger: env.logger,
-         scheduler: env.scheduler,
-       }),
-     );
-     /* highlight-add-end */
-
-     const { processingEngine, router } = await builder.build();
-     await processingEngine.start();
-     return router;
-   }
-   ```
+2. Configure the scheduler for the entity provider using one of the following methods:
+
+   - **Method 1**: If the scheduler is configured inside the `app-config.yaml` file, add the following code to `packages/backend/src/plugins/catalog.ts` file:
+
+     ```ts title="packages/backend/src/plugins/catalog.ts"
+     /* highlight-add-next-line */
+     import { ThreeScaleApiEntityProvider } from '@janus-idp/backstage-plugin-3scale-backend';
+
+     export default async function createPlugin(
+       env: PluginEnvironment,
+     ): Promise<Router> {
+       const builder = await CatalogBuilder.create(env);
+
+       /* ... other processors and/or providers ... */
+       /* highlight-add-start */
+       builder.addEntityProvider(
+         ThreeScaleApiEntityProvider.fromConfig(env.config, {
+           logger: env.logger,
+           scheduler: env.scheduler,
+         }),
+       );
+       /* highlight-add-end */
+
+       const { processingEngine, router } = await builder.build();
+       await processingEngine.start();
+       return router;
+     }
+     ```
+
+     ***
+
+     **NOTE**
+
+     If you have made any changes to the schedule in the `app-config.yaml` file, then restart to apply the changes.
+
+     ***
+
+   - **Method 2**: Add a schedule directly inside the `packages/backend/src/plugins/catalog.ts` file as follows:
+
+     ```ts title="packages/backend/src/plugins/catalog.ts"
+     /* highlight-add-next-line */
+     import { ThreeScaleApiEntityProvider } from '@janus-idp/backstage-plugin-3scale-backend';
+
+     export default async function createPlugin(
+       env: PluginEnvironment,
+     ): Promise<Router> {
+       const builder = await CatalogBuilder.create(env);
+
+       /* ... other processors and/or providers ... */
+       /* highlight-add-start */
+       builder.addEntityProvider(
+         ThreeScaleApiEntityProvider.fromConfig(env.config, {
+           logger: env.logger,
+           schedule: env.scheduler.createScheduledTaskRunner({
+             frequency: { minutes: 1 },
+             timeout: { minutes: 1 },
+           }),
+         }),
+       );
+       /* highlight-add-end */
+
+       const { processingEngine, router } = await builder.build();
+       await processingEngine.start();
+       return router;
+     }
+     ```
+
+   - If both the `schedule` (hard-coded schedule) and `scheduler` (`app-config.yaml` schedule) option are provided in the `packages/backend/src/plugins/catalog.ts`, the `scheduler` option takes precedence.
+
+     - If the schedule inside the `app-config.yaml` file is not configured, then the `schedule` option is used.
```

Comment:
I do not understand why is this here. You're within a section that says "both are defined", so why backtrack to the "Method 2" in this sentence?

Reply:
I mainly wanted to mention the expected behavior when both are `schedule` and `scheduler` are defined.